test/e2e/testdata/images/tmp-writer/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM mikefarah/yq:4.34.1
+
+COPY writer.sh /bin
+USER root
+RUN chmod +x /bin/writer.sh
```

Comment:
you can avoid this line if you set the proper file permission (+x) on `write.sh` within git repo.

Reply:
You can also set them from the COPY command directly via the chmod flag: https://docs.docker.com/engine/reference/builder/#copy

Reply:
Thanks guys, I extended the `COPY` command with `chmod`. I like explicitly showing the file permission.